WordPressAuthenticator/GoogleSignIn/OAuthError.swift

```diff
@@ -5,12 +5,20 @@ enum OAuthError: LocalizedError {
 
     case failedToGenerateSecureRandomCodeVerifier(status: Int32)
 
+    // OAuth token response
+    case urlDidNotContainCodeParameter(url: URL)
+    case tokenResponseDidNotIncludeIdToken
+
     var errorDescription: String {
```

Comment:
We probably should make the `OAuthError` public, so that when the app captures the error thrown by the `NewGoogleAuthenticator.getOAuthToken`, they cast the error to the correct type and handle it accordingly.

Reply:
Good point. I addressed it in 3c49abdd.

> so that when the app captures the error thrown by the `NewGoogleAuthenticator.getOAuthToken`

Worth pointing out that, for the time being, no app will call this directly. Still, it's handy to be ready for when / if they'll want to.